docs/resources/azure_network_watchers.md.erb

```diff
@@ -0,0 +1,63 @@
+---
+title: About the azure_network_watchers Resource
+platform: azure
+---
+
+# azure\_network\_watchers
+
+Use the `azure_network_watchers` InSpec audit resource to verify that a Network Watcher exists.
+
+<br />
+
+## Syntax
+
+An `azure_network_watchers` resource block identifies Network Watchers by Resource Group.
+
+    describe azure_network_watchers(resource_group: 'example_group') do
+      ...
+    end
+<br />
+
+## Examples
+
+### Test that an example Resource Group has the named Network Watcher
+
+    describe azure_network_watchers(resource_group: 'example_group') do
+      its('names') { should include('ExampleNetworkWatcher') }
+    end
+
+<br />
+
+## Properties
+
+* `names`
+
+### names
+
+The name of the Network Watcher
+
+    its('names') { should include('ExampleNetworkWatcher') }
+
+## Matchers
+
+This InSpec audit resource has the following special matchers. For a full list
+of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### exists
+
+The control will pass if the resource returns a result. Use `should_not` if you expect zero matches.
+
+    # If we expect 'example_group' Resource Group to have Network Watchers
+    describe azure_network_watchers(resource_group: 'example_group') do
```

Comment:
We should use mixed case instead of snake case. `ExampleGroup` would be my recommendation.